### ISSUE TITLE
Request for checkpoint to improve poor final results

### ISSUE THREAD

**Issue by 1149722739** [agiresearch/OpenP5#11]
Hello,

I have been working on implementing your code in my project but unfortunately the final results are quite poor. I suspect there may be issues with how I am using the code.

So first,I wanted to ask about the parameter settings you used to achieve the results in your paper.I have tried tweaking the batch size, learning rate, and number of epochs but can't seem to find the right combination of parameters. Do you have any guidance on the optimal parameter ranges or settings you used for your experiments?

Second,I noticed some issues in the code. I tried to fix the issues, but wanted to check with you on the proper way to handle them.

Third,would it be possible to get a checkpoint or snapshot of the model after training?Having a checkpoint would really help me debug my implementation and understand what I'm doing wrong. If there are any examples or guides on proper usage that could also be helpful. My end goal is to reproduce the results in your paper as closely as possible.

Thank you for creating this useful code - I look forward to hearing back from you!

**Comment by shuyuan-x** [comment]
Hi,

Thanks for your interest in our work. The results in the paper is obtained from the original version of the project which is put into "release-1.0" branch. We have made significant modification to make T5 and LLaMa compatible in the same project. We are running checkpoints based on current implementation and will release them once we got results.

However, as mentioned above, the release-1.0 branch is a complete and readily executable branch that can help you to quickly get things running and do experiments. We also included checkpoints in the release-1.0 branch. If you have any questions regarding the issues in the code, you can also directly email me.

Hope this is helpful to your research, and thanks again for your interest in this work.
